### DISCUSSION TITLE
Molex pinout for opengenset with power pact generator?

### DISCUSSION THREAD

**Discussion by Nate52441** [jgyates/genmon#1210] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

N/A

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I’m working on setting up my opengenset Molex connector with my generic power pact with evolution 1.0 controller and want to make sure I’m understanding the pinout correct for end to end setup. Can someone double check my logic?

opengenset.               Powerpact

Pin 1 (GND)————–Pin 2
Pin 2 (+5v)—————Pin 1
Pin 5 (RX)—————–Pin 8 (TX)
Pin 6 (TX)—————–Pin 7 (RX) 

and those would be the only cables I’ll have connected since opengenset can run off +5v.

**Comment by jgyates** [comment]
The pinout is on this page: https://github.com/jgyates/genmon/wiki/3.1--Making-a-Cable

**Comment by skipfire** [comment]
I also replied to your contact request with the below information (I'm the main guy behind PintSize), but posting here for anyone who stumbles by later.

OpenGenSet can be powered by the 12v or 5v pins on the Molex or by the USB-C port.  On OpenGenSet, HATs, pHATs, and the USB adapter the top two pins on the right are the data pins and the bottom row is 5v (usable by OGS and USB products), ground, 12v and the floating ground (not connected on OGS).

**Comment by skipfire** [comment]
@jgyates can appendix R be deleted?  There is no need for custom cabling as the PowerPact controller has the same pinout as the other supported Generacs.

**Reply by jgyates** [reply]
I think so. I will do that later today.

**Reply by jgyates** [reply]
done.

**Comment by Nate52441** [comment]
Dropping some wiring pictures here incase someone needs help in the future, the standard cable from Pintsize would have worked  but I removed unused cables and repinned due to looking at the diagrams incorrectly. 


Opengenset side 
![IMG_2029](https://github.com/user-attachments/assets/aaf7e269-a1c7-4e0e-b323-2073920a6d6a)

Controller side 

![IMG_2026](https://github.com/user-attachments/assets/44b77d7f-6762-4730-a18a-5279d3b99a67)

 Powerpact controller for reference 

![IMG_2025](https://github.com/user-attachments/assets/91f2d38d-e295-4f80-9b33-89da4e8942bb)
